parameters.cpp

```diff
@@ -676,6 +681,10 @@ void Parameters::getParameters() {
    RP::get("vlasovsolver.maxSlAccelerationSubcycles", P::maxSlAccelerationSubcycles);
    RP::get("vlasovsolver.maxCFL", P::vlasovSolverMaxCFL);
    RP::get("vlasovsolver.minCFL", P::vlasovSolverMinCFL);
+   RP::get("vlasovsolver.LocalTranslate",P::vlasovSolverLocalTranslate);
+   if ((myRank == MASTER_RANK)&&(P::vlasovSolverLocalTranslate==true)) {
+      printf("Performing all spatial translation locally using ghost cell information\n");
```

Comment:
This could go to logfile.

Reply:
Perhaps indeed. We should then also consider the outputs of AMR filtering - perhaps they should also go there?

Reply:
I think so too. Parsing what system output file printed what line at what time w.r.t. the logfile is becoming increasingly complex, in other branches (e.g. ionosphere) I have also started printing more to the logfile so we have everything in one place in chronological order.